project/webapp/templates/partials/thread/thread_body.html

```diff
@@ -41,6 +41,144 @@
                             <span class="text">Export</span>
                         </a>
                     </div>
+                    <div class="add-button" style="margin-left: 4px; position: relative">
```

Comment:
See if you can encapsulate this into a `share_buttons` partial template, and include it here. That way it is reusable, and keeps the top level threads template relatively clean.